RoboFile.php

```diff
@@ -17,9 +17,32 @@ class RoboFile extends Tasks implements LoggerAwareInterface {
 
   use LoggerAwareTrait;
 
+  /**
+   * BLT root.
```

Comment:
Oh really, a comment is required for each var? I feel that this is a pretty bad habit, these usually just say the same thing as the var name.

Reply:
For now, yeah they are mostly just the variable name because there are _so_ many and I don't have time to document them better. Long-term, they should be documented more thoroughly.

Unfortunately we don't have much control over this. This is the [Drupal code comment standard](https://www.drupal.org/node/1354#var). I've looked and we can't disable it without disabling the requirement for _all_ short descriptions, or else all variable comments (including type hinting).